src/common/components/Tag/Tag.theme.ts

```diff
@@ -0,0 +1,50 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type TagVariants = VariantProps<typeof tagTheme>;
+
+export const tagTheme = tv(
+  {
+    slots: {
+      tag: [
+        "inline-flex",
+        "min-w-14",
+        "py-[0.125rem]",
+        "px-3",
+        "gap-2",
+        "justify-center",
+        "items-center",
+        "rounded-[6.1875rem]",
+        "border",
+        "border-solid",
+        "border-border-default",
+        "transition-color",
+        "duration-300",
+        "ease-in-out",
+        "bg-transparent",
+        "hover:border-border-secondary",

```

Comment:
If we are using Tags as read only, do we still need hover states? I'm thinking we might just need the default and active states 🤔

Reply:
removed at 46ab314!